packages/loader/driver-utils/src/network.ts

```diff
@@ -43,8 +43,11 @@ export class GenericNetworkError extends LoggingError implements IDriverErrorBas
     }
 }
 
-export class DeltaStreamConnectionForbiddenError extends LoggingError implements IDriverErrorBase {
-    readonly errorType = DriverErrorType.deltaStreamConnectionForbidden;
+const deltaStreamConnectionForbiddenStr = "deltaStreamConnectionForbidden";
+export class DeltaStreamConnectionForbiddenError extends LoggingError {
+    static readonly errorType: string =
+        DriverErrorType[deltaStreamConnectionForbiddenStr] ?? deltaStreamConnectionForbiddenStr;
```

Comment:
I'm not sure I follow why you need ?? part here. Note that due to dynamic driver loading, loader may use different versions of ODSP driver, and if different versions of driver use different codes, we are screwed.
I.e. this code (and all other errors codes) can't change over time, and thus they are part of driver-definitions. This change moves it out, i.e. now we loader uses this class instance to get error code that should never change.

Reply:
i'll add comments, but think of this as forward compat for new error messages